### ISSUE TITLE
Add examples to documentation

### ISSUE THREAD

**Issue by NullSense** [MrCappuccino/Linda#12]
Examples need to be added to the public items as per [Rust documentation guidelines](https://rust-lang-nursery.github.io/api-guidelines/documentation.html).

**Comment by Rajratnpranesh** [comment]
I want to contribute to the repo. Please discuss the details of the issue.
Thanks

**Comment by NullSense** [comment]
@Rajratnpranesh Thanks for taking up this issue, basically examples of how to use public API functions need to be added to the documentation. The [Rust documentation guidelines](https://rust-lang-nursery.github.io/api-guidelines/documentation.html) are a good place to start, you might also want to take a look at other crates, e.g. the [http crate request.rs docs](https://docs.rs/http/0.1.19/src/http/request.rs.html#70).

**Comment by Rajratnpranesh** [comment]
@NullSense Can it be any API function that can be used as an example?

**Comment by NullSense** [comment]
Yes, there's only a few public ones, feel free to document all of them if you like.

**Comment by Rajratnpranesh** [comment]
@NullSense I made a pull request to add **API_Documentation.md**. Please have a look. I talked about calling GitHub Public API.

**Comment by NullSense** [comment]
@Rajratnpranesh Sorry if I didn't make the issue clear, but by documentation of the public facing API, I meant documentation for the Linda project itself not GitHub.
